src/kem/kyber/serialize.rs

```diff
@@ -1,35 +1,25 @@
 use super::{
-    arithmetic::{to_unsigned_representative, KyberFieldElement, KyberPolynomialRingElement},
-    compress::{compress_q, decompress_q},
+    arithmetic::{to_unsigned_representative, PolynomialRingElement, StandardFieldElement},
+    compress::{
+        compress_ciphertext_coefficient, compress_message_coefficient,
+        decompress_ciphertext_coefficient, decompress_message_coefficient,
+    },
     constants::{BYTES_PER_RING_ELEMENT, COEFFICIENTS_IN_RING_ELEMENT, SHARED_SECRET_SIZE},
```

Comment:
`COEFFICIENTS_IN_RING_ELEMENT` seems unused? It's only used in hax debug asserts. So either only include it for hax, or use the qualified path to avoid the warning.

Reply:
Added a `cfg` guard.